src/components/files-grid-viewer.less

```diff
@@ -0,0 +1,42 @@
+@files-grid-viewer-file-icon-padding: 10px;
+@files-grid-viewer-file-container-base-width: 60px;
+@files-grid-viewer-file-container-total-width: @files-grid-viewer-file-container-base-width + 2 * @files-grid-viewer-file-icon-padding;
+
+.FilesGridViewer {
+  display: flex;
+  gap: 20px;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: flex-start;
+  padding: @edu-content-padding;
+  border: 1px solid @edu-border-color-base;
```

Comment:
`text-align: center;` here would look better (in case of the short "..."), what dou you think?